sky/cli.py

```diff
@@ -765,6 +765,8 @@ def _make_task_or_dag_from_entrypoint_with_overrides(
                     f'WARNING: override params {override_params} are ignored, '
                     'since the yaml file contains multiple tasks.',
                     fg='yellow')
+            for task in dag.tasks:
+                task.update_envs(env)
```

Comment:
Is this required after removing L93-L98 in `dag_utils.py::load_chain_dag_from_yaml`? 

I may be missing something, but it seems like the `dag = dag_utils.load_chain_dag_from_yaml(entrypoint, env_overrides=env)` call at L758 will now take care of overriding env vars for each task and this should not be required?

Reply:
Good point! Fixed. Thanks!